pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -263,14 +263,14 @@ spec:
           failureThreshold: 10
         livenessProbe:
           tcpSocket:
-            port: tcp-health-port
+            port: 3333
```

Comment:
I think we should continue to use port naming for this, and change the portion of the template near line 234 where we say "if port == 3333 { this is the health port}

Reply:
done